mdtools.py

```diff
@@ -48,46 +58,57 @@ def mdrun(
     check_point_file: str = None,
     mdrun_options: list = None,
     gpus_per_node: int = None,
+    plumed_file: str = None,
+    use_api: bool = True
 ):
-    mpi_rank = max(mpi_rank - 1, 0)
+    mpi_rank = mpi_rank - 1
     cwd = os.path.abspath(os.getcwd())
     os.chdir(output_dir)
     input_files = {"-s": tpr_file}
     if check_point_file is not None:
         input_files["-cpi"] = check_point_file
+    if plumed_file is not None:
+        input_files["-plumed"] = plumed_file
     # SPC increased state printing to every 5 minutes since swarms are short
-    if mdrun_options is None:
+    if mdrun_options is None:  # TODO
         mdrun_options_parse = []
     else:
-        mdrun_options_parse = mdrun_options[:]
+        mdrun_options_parse = mdrun_options
```

Comment:
Not adding this generated a bug.
```suggestion
        mdrun_options_parse = mdrun_options[:]
```